### PR TITLE
Adds support for phpunit 7

### PR DESCRIPTION
**Pull request by kimpepper**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   ],
   "require": {
     "php": "^7.1",
-    "phpunit/phpunit": "^6.5",
+    "phpunit/phpunit": "^7.5",
```
Comment (larowlan) [inline composer.json]: I think this should be as per #2 i.e. `^6.5 || ~7.0`

**Review by larowlan — COMMENTED** [review]
I think we should make it either/or like #2

File: composer.json (hunk 1 above)
Comment (kimpepper) [reply composer.json]: Done

**Comment by larowlan** [comment]
pushed my WIP test

**Pull request merged**